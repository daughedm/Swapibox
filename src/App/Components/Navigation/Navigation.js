import React from 'react';
import Button from '../Button/Button';
import '../Navigation/Navigation.css';
import StarWarsLogo from '../../../Assets/Images/Star_Wars.svg';

const Navigation = (props) => {

  return(
    <div className="nav-container" >
      <img className="logo" src={StarWarsLogo} alt="star wars logo" />
      <Button />
    </div>
  )
}

export default Navigation;