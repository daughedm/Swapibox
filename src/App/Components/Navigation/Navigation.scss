$sw-yellow: rgb(246, 231, 48);

* {
  z-index: 3;
}

.nav-container {
  height: 170px;
  width: 100%;
  display: flex;
  background-image: url(../../../Assets/Images/star-bg.jpg);
  background-repeat: no-repeat;
  background-size: cover;
  position: fixed;
  border-bottom: 3px solid $sw-yellow;
  justify-content: space-between;
  
}

.logo {
  width: 220px;
  padding: 0 80px;
}

