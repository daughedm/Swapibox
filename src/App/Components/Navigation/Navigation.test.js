import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import Navigation from './Navigation';

describe('Main', () => {
  let wrapper;

  it('should match snapshot', () => {
    const favorites = ['favorite'];
    const handleClick = jest.fn();

    wrapper = shallow(<Navigation
      category="people"
      handleClick={handleClick}
      favorites={favorites} />)

    expect(wrapper).toMatchSnapshot();
  });
})
