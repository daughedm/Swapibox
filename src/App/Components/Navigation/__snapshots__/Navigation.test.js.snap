// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Navigation
    category="people"
    favorites={
        Array [
            "favorite",
          ]
    }
    handleClick={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          alt="star wars logo"
          className="logo"
          src="Star_Wars.svg"
/>,
        <Button
          favorites={
                    Array [
                              "favorite",
                            ]
          }
          handleClick={[Function]}
/>,
      ],
      "className": "nav-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "star wars logo",
          "className": "logo",
          "src": "Star_Wars.svg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "favorites": Array [
            "favorite",
          ],
          "handleClick": [Function],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="star wars logo"
            className="logo"
            src="Star_Wars.svg"
/>,
          <Button
            favorites={
                        Array [
                                    "favorite",
                                  ]
            }
            handleClick={[Function]}
/>,
        ],
        "className": "nav-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "star wars logo",
            "className": "logo",
            "src": "Star_Wars.svg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "favorites": Array [
              "favorite",
            ],
            "handleClick": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
